recipes/xtrabackuprb.rb

```diff
@@ -1,5 +1,12 @@
 include_recipe 'git'
 include_recipe 'percona::package_repo'
+include_recipe 'yum-epel'
+
+package 'rubygems' do
+end.run_action(:install)
```

Comment:
This isn't needed since we're using the chef rubygems.

Reply:
This is an error I made because I accidentally make `chef_gem 'xtrabackup-rb'` to compile_time false, went down the rabbit hole and installed rubygems. It is now corrected.

Reply:
However, we will need 'libev' package for 'percona-xtrabackup', so still need to include_recipe 'yum-epel' for installing 'libev'